examples/generated/test/appkit/images/LocalAsset.swift

```diff
@@ -46,7 +46,8 @@ public class LocalAsset: NSBox {
 
     addSubview(imageView)
 
-    imageView.fillColor = #colorLiteral(red: 0.847058823529, green: 0.847058823529, blue: 0.847058823529, alpha: 1)
+    // FIXME: Value of type 'NSImageView' has no member 'fillColor'
+//    imageView.fillColor = #colorLiteral(red: 0.847058823529, green: 0.847058823529, blue: 0.847058823529, alpha: 1)
```

Comment:
@dabbott Looks like there is some issue with generating `fillColor` here (run snapshot to see it), so I manually commented this line to allow the project to compile. 

Fixing this is outside of the scope of this PR.

Reply:
Good catch! Hmm... yeah, might have to get creative with NSImageView in that case. For another PR though.